runtime/common/src/lib.rs

```diff
@@ -30,6 +30,8 @@ pub const MINUTES: BlockNumber = 60_000 / (MILLISECS_PER_BLOCK as BlockNumber);
 pub const HOURS: BlockNumber = MINUTES * 60;
 pub const DAYS: BlockNumber = HOURS * 24;
 
+pub const UNITS: Balance = 10_000_000_000;
```

Comment:
Is this used somewhere ?

Reply:
No, it was used before but forgot to remove it. Thanks!

Addressed in [bea00b4](https://github.com/paritytech/extended-parachain-template/pull/94/commits/bea00b40c158499dc1c86c2bbcae517406782b5c).